### PR TITLE
Add required packages, adopt example for the browser

### PR DESCRIPTION
**Pull request by vishniakov-nikolai**

### CODE REVIEW

File: http-server.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+const httpServer = require('http-server');
+
+const root = path.join(__dirname, '.');
+const port = process.env['PORT'] || 8081;
+const host = '127.0.0.1';
```
Comment (riverlijunjie) [inline http-server.js:6]: Can we setup httpServer for remote access?
It seems https should be applied due to 'Cross-Origin-Opener-Policy' and 'Cross-Origin-Embedder-Policy', but https requires SSL certificate.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,285 @@
+{
+  "name": "wasm_simple_sample",
```
Comment (riverlijunjie) [inline package-lock.json:2]: This json file is generated by npm?

**Review by riverlijunjie — COMMENTED** [review]
Thank you provide the method how to run JS both on local and browser, it is very helpful for me!

File: http-server.js (hunk 1 above)
Comment (vishniakov-nikolai) [reply http-server.js:6]: You can run server script on the remote machine and open ssh-tunnel to keep this connection like local.
Or do you prefer run this script on domain?

File: package-lock.json (hunk 2 above)
Comment (vishniakov-nikolai) [reply package-lock.json:2]: Yes, correct. This file created automatically.

**Pull request merged**